README.md

```diff
@@ -202,15 +202,25 @@ poetry run flask drop-db
 
 ## Migrations
 
+If you have added new mapped dataclasses or modified existing ones, a migration script needs to be added.
+This script updates the tables and columns of the database to match the mapped dataclasses.
+To generate the migration script you need to do the following steps:
+
 ```bash
-# create a new migration after changes in the db (Always manually review the created migration!)
-poetry run flask db migrate -m "Initial migration."
-# upgrade db to the newest migration
+# delete the content of the database
```

Comment:
Just delete the whole DB (as drop-db will not drop the metadata table that is created when the database was made from a migration and not through the create-db command).